src/test/java/com/feedzai/commons/sql/abstraction/engine/EngineTestUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2014 Feedzai
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.feedzai.commons.sql.abstraction.engine;
+
+import com.feedzai.commons.sql.abstraction.ddl.DbEntity;
+
+import static com.feedzai.commons.sql.abstraction.ddl.DbColumnType.*;
+import static com.feedzai.commons.sql.abstraction.dml.dialect.SqlBuilder.dbEntity;
+
+/**
+ * Utils for using in {@link DatabaseEngine} tests.
+ *
+ * @author David Fialho (david.fialho@feedzai.com)
+ * @since 2.1.13
+ */
+public interface EngineTestUtils {
```

Comment:
Any reason why you chose an interface instead of a class?

Reply:
I used an interface because this is supposed to include static methods. As far as I know, using a class for utilities just adds unnecessary boilerplate code. I might be wrong, though. Is there any reason to use a class instead of an interface for this purpose?

Reply:
That boilerplate code would just be a private constructor to prevent instantiation: nothing else. Let me put it this way: do you imagine that interface be implemented by several classes? Or are you thinking that in the future we are going to leverage polymorphism and use implementations of EngineTestUtil?

If you answer yes to either question, that perhaps my judgment is wrong. 

Otherwise, from the name of the interface and its comments, you are creating something to helps you do other stuff which is not directly related to "objects" and "polymorphism". In that case, you don't want instantiation (and therefore you don't want to create objects). Right now I can to that by implementing that interface. For a "true" utility "class", you could just create a "public final class EngineTestUtils" and create a private constructor (which will prevent creating instances or extend that class). 

Therefore, the changes would be:
- change "interface" to "final class"
- create private constructor
- make that static method public.

Also, I think that you still have imports with "*". Could you please take a look at that?

Reply:
You make a good point about a subclass/implementation still being able to be instantiated.
I'll make the proposed changes. 

Thank you for taking the time to answer this thoroughly.